src/common/playlists/PlayHeaderInfo.jsx

```diff
@@ -40,7 +40,11 @@ const Tags = ({tags}) => {
 };
 
 const PlayHeaderInfo = ({ play }) => {
-   
+  
+  const structureTags = play?.play_tags.reduce((pre, next) => {
```

Comment:
@Angryman18 shouldn't this go inside the `Tags` component instead as the logic belongs to the tag and we pass the play.play_tags to the `Tags` component as the prop?

Reply:
this is fixed